services/galley/src/Galley/API/Create.hs

```diff
@@ -84,7 +84,6 @@ createTeamGroupConv zusr zcon tinfo body = do
             let uu = filter (/= zusr) $ map (view userId) mems
             checkedConvSize uu
         else do
-            void $ permissionCheck zusr AddRemoveConvMember mems
```

Comment:
Let's change this into sth like: _if user has no AddConvMember permission, then ensure size <= 1_